docs/src/building-from-source.md

```diff
@@ -1,160 +0,0 @@
-$ git checkout $TAG
-```
-
-### Configuration Setup
-
-Ensure important programs such as the vote program are built before any nodes are started. Note that we are using the release build here for good performance.
-If you want the debug build, use just `cargo build` and omit the `NDEBUG=1` part of the command.
-
-```bash
-$ cargo build --release
-```
-
-The network is initialized with a genesis ledger generated by running the following script.
-
-```bash
-$ NDEBUG=1 ./multinode-demo/setup.sh
-```
-
-### Drone
-
-In order for the validators and clients to work, we'll need to spin up a faucet to give out some test tokens. The faucet delivers Milton Friedman-style "air drops" \(free tokens to requesting clients\) to be used in test transactions.
-
-Start the faucet with:
-
-```bash
-$ NDEBUG=1 ./multinode-demo/faucet.sh
-```
-
-### Singlenode Testnet
-
-Before you start a validator, make sure you know the IP address of the machine you want to be the bootstrap validator for the demo, and make sure that udp ports 8000-10000 are open on all the machines you want to test with.
-
-Now start the bootstrap validator in a separate shell:
-
-```bash
-$ NDEBUG=1 ./multinode-demo/bootstrap-validator.sh
-```
-
-Wait a few seconds for the server to initialize. It will print "leader ready..." when it's ready to receive transactions. The leader will request some tokens from the faucet if it doesn't have any. The faucet does not need to be running for subsequent leader starts.
-
-### Multinode Testnet
-
-To run a multinode testnet, after starting a leader node, spin up some additional validators in separate shells:
-
-```bash
-$ NDEBUG=1 ./multinode-demo/validator-x.sh
-```
-
-To run a performance-enhanced validator on Linux, [CUDA 10.0](https://developer.nvidia.com/cuda-downloads) must be installed on your system:
-
-```bash
-$ ./fetch-perf-libs.sh
-$ NDEBUG=1 SOLANA_CUDA=1 ./multinode-demo/bootstrap-validator.sh
-$ NDEBUG=1 SOLANA_CUDA=1 ./multinode-demo/validator.sh
-```
-
-### Testnet Client Demo
-
-Now that your singlenode or multinode testnet is up and running let's send it some transactions!
-
-In a separate shell start the client:
-
-```bash
-$ NDEBUG=1 ./multinode-demo/bench-tps.sh # runs against localhost by default
-```
-
-What just happened? The client demo spins up several threads to send 500,000 transactions to the testnet as quickly as it can. The client then pings the testnet periodically to see how many transactions it processed in that time. Take note that the demo intentionally floods the network with UDP packets, such that the network will almost certainly drop a bunch of them. This ensures the testnet has an opportunity to reach 710k TPS. The client demo completes after it has convinced itself the testnet won't process any additional transactions. You should see several TPS measurements printed to the screen. In the multinode variation, you'll see TPS measurements for each validator node as well.
```

Comment:
TODO: Gotta much sure docs like this aren't lost

Reply:
a simple process: move such text to a section at the end of the doc, possibly with a tiny remark "move to another location".